packages/core/forms/src/generator/fields/core/fieldLabel.vue

```diff
@@ -1,17 +1,51 @@
-<template lang="pug">
-span(:id="getFieldID(schema)", :class="schema.fieldClasses", v-attributes="'label'") {{ value }}
+<template>
+  <span
+    class="field-label"
+    :class="schema.fieldClasses"
+  >
+    <KLabel
+      :id="getFieldID(schema)"
+      v-attributes="'label'"
+    >{{ labelValue }}</KLabel>
+  </span>
 </template>
 
-<script>
-import abstractField from '../abstractField'
+<script lang="ts" setup>
+import type { PropType } from 'vue'
+import composables from '../../../composables'
 
-export default {
-  mixins: [abstractField],
-}
+const props = defineProps({
+  disabled: {
+    type: Boolean,
+    default: false,
+  },
+  formOptions: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => undefined,
+  },
+  model: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => undefined,
+  },
+  schema: {
+    type: Object as PropType<Record<string, any>>,
+    required: true,
+  },
+  vfg: {
+    type: Object,
+    required: true,
+  },
+})
```

Comment:
Curious: Would we prefer `withDefaults(defineProps<T>(), ...)` when there is no need for validators?